backend/lcfs/web/api/compliance_report/schema.py

```diff
@@ -113,7 +113,7 @@ class ComplianceReportBaseSchema(BaseSchema):
     # transaction: Optional[TransactionBaseSchema] = None
     nickname: Optional[str] = None
     supplemental_note: Optional[str] = None
-    update_date: Optional[datetime] = None
+    # update_date: Optional[datetime] = None
```

Comment:
this will impacting latest status update column in compliance report list table.
![image](https://github.com/user-attachments/assets/6465a709-7090-424b-9090-5b2b6fb8e11d)

Reply:
fixed.